src/umpire/resource/DetectVendor.hpp

```diff
@@ -0,0 +1,28 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Can you make this file snake case? `detect_vendor.hpp` - I've been trying to follow that if the file doesn't contain a class.